src/main/java/org/prebid/server/bidder/interactiveoffers/InteractiveOffersBidder.java

```diff
@@ -0,0 +1,131 @@
+package org.prebid.server.bidder.interactiveoffers;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import io.vertx.core.http.HttpMethod;
+import org.apache.commons.collections4.CollectionUtils;
+import org.prebid.server.bidder.Bidder;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpCall;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.json.DecodeException;
+import org.prebid.server.json.JacksonMapper;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.interactiveoffers.ExtImpInteractiveoffers;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class InteractiveOffersBidder implements Bidder<BidRequest> {
+
+    private static final TypeReference<ExtPrebid<?, ExtImpInteractiveoffers>> INTERACTIVEOFFERS_EXT_TYPE_REFERENCE =
+            new TypeReference<ExtPrebid<?, ExtImpInteractiveoffers>>() {
+            };
+
+    private final String endpointUrl;
+    private final JacksonMapper mapper;
+
+    public InteractiveOffersBidder(String endpointUrl, JacksonMapper mapper) {
+        this.endpointUrl = HttpUtil.validateUrl(Objects.requireNonNull(endpointUrl));
+        this.mapper = mapper;
+    }
+
+    @Override
+    public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest request) {
+        final List<Imp> modifiedImps = new ArrayList<>();
+
+        for (Imp imp : request.getImp()) {
+            try {
+                final ExtImpInteractiveoffers impExt = parseImpExt(imp);
+
+                if (impExt.getPubId() == null) {
+                    throw new PreBidException("The pubid must be present");
+                }
+
+                modifiedImps.add(imp.toBuilder().ext(impExtToObjectNode(impExt)).build());
+            } catch (PreBidException e) {
+                return Result.withError(BidderError.badInput(e.getMessage()));
+            }
+        }
```

Comment:
We don't need this, they don't do it in go, and also server will not process invalid ext request because of json validation schema